src/App.js

```diff
@@ -92,6 +93,7 @@ function App() {
       setExpenses(expenses.filter(expense => expense.id !== id));
       if (filteredExpenses.length === 1) setSelectedMonth(currentMonth);
       setExpenseDeleted(true);
+      setTimeout(() => setExpenseDeleted(undefined), 5000);
```

Comment:
i am not sure if I get the purpose of this delayed action. `setExpenseAdded/Deleted(undefined)` does not communicate its purpose.

Reply:
I added a comment 👍

Reply:
nice. thanks for the comment. did you consider renaming stuff?

Reply:
you mean renaming functions? I would consider it, yes. Let me think about it...
I just did not do it because in `App.js` line 25-26 there are already some comments and clarifications from Matt regarding the usage of these setters and states.

Reply:
removed comments and renamed functions